BlazarUI/app/scripts/components/shared/PageHeader.jsx

```diff
@@ -1,6 +1,19 @@
 import React, {Component, PropTypes} from 'react';
+import $ from 'jquery';
```

Comment:
i've honestly done little with react animations, but take a look at -> http://unitstep.net/blog/2015/03/03/using-react-animations-to-transition-between-ui-states this might do the trick